tests/export/html/test_markup_compatibility.py

```diff
@@ -91,3 +91,71 @@ def test_textbox_with_content_outside_of_textbox(self):
             </p>
         '''
         self.assert_document_generates_html(document, expected_html)
+
+    def test_fallback_is_only_text(self):
+        document_xml = '''
+            <p>
+                <r><t>AAA</t></r>
+                <r>
+                    <t>BBB</t>
+                    <AlternateContent>
+                        <Fallback>
+                            <r>
+                                <t>CCC</t>
+                            </r>
+                        </Fallback>
+                    </AlternateContent>
+                    <t>DDD</t>
+                </r>
+                <r><t>EEE</t></r>
+            </p>
+        '''
+
+        document = WordprocessingDocumentFactory()
+        document.add(MainDocumentPart, document_xml)
+
+        expected_html = '''
+            <p>AAABBBCCCDDDEEE</p>
+        '''
+        self.assert_document_generates_html(document, expected_html)
+
+    def test_fallback_is_a_table(self):
+        document_xml = '''
+            <p>
+                <r><t>AAA</t></r>
+                <r>
+                    <t>BBB</t>
+                    <AlternateContent>
+                        <Fallback>
+                            <tbl>
```

Comment:
Alright, I have most of this actually figured out. The only pickle is the valid children of the `Fallback` element. Which is actually odd now that I think about it. Since we are should be removing the `Fallback` element and the re-parse should do everything we need. So perhaps I'm missing something? Or I did it wrong? Not sure.